webui/src/pages/Home.js

```diff
@@ -11,6 +12,7 @@ const Home = () => {
       <div style={{ display: 'flex', flexDirection: 'row', marginTop: 20 }}>
         <PoolCreator />
         <TokenSwapper selectedPair={pair} />
```

Comment:
Now that Swap component is partly functional, can TokenSwapper be removed?

Reply:
TokenSwapper can be removed now, I'll do it right now.